### ISSUE TITLE
Method counter_clockwise in volume.py does not do what the docstring says.

### ISSUE THREAD

**Issue by B-Hartmann** [ZELLMECHANIK-DRESDEN/dclab#141]
The method counter_clockwise() in the file dclab/features/volume is supposed to return contour coordinates in a counter-clockwise order, yet the returned coordinates are in clockwise order.

minimal working example:

```
import numpy as np
cx = np.array([+1, +1, -1, -1])
cy = np.array([+1, -1, -1, +1])

angles = np.unwrap(np.arctan2(cy, cx))
grad = np.gradient(angles)
if np.average(grad) > 0:
    x = cx[::-1]
    y = cy[::-1]
else:
    x = cx
    y = cy
print(x)
print(y)
```

gives the output: 
[ 1  1 -1 -1]
[ 1 -1 -1  1]

**Comment by paulmueller** [comment]
After comparing with the original matlab script, It also appears as if this line:

https://github.com/ZELLMECHANIK-DRESDEN/dclab/blob/aade02014f4807053b2c2a668474dcdcdca9588a/dclab/features/volume.py#L152

Is actually supposed to read

```python
v3 = -1 * d_r**2 * z_vec_m1
```

I'm afraid this calls for thorough testing.

**Comment by maikherbig** [comment]
Hi,
assuming ShapeOut and dclab are limited (so)RT(F)DC data, this is not a problem since the contours, returned by cv2.findContours (in ShapeIn) will always be counterclockwise (because only the outer contour is returned -> cv2.retr_external). In case, ShapeIn get an update where it starts to return inner contours too (e.g. for assessing the shape of bubbles or the nucleus inside cells) it would actually cause a problem as OpenCV returns clockwise inner contours. However, computing the volume for inner contours does not really make sense as rotational symmetry cannot be assumed.
In conclusion, one could either simply delete line 85
https://github.com/ZELLMECHANIK-DRESDEN/dclab/blob/97f9f4ebeb84fd6f7cf7d239874a8675b5d90feb/dclab/features/volume.py#L85
or fix the function def counter_clockwise(cx, cy), or insert a test if a contour is counter-clockwise (return np.nan if not).
